modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -44,6 +44,9 @@ def submit_form_526 # rubocop:disable Metrics/MethodLength
             source: source_name
           )
 
+          ClaimsApi::Logger.log('526', claim_id: params[:id], detail: 'Submitted',
```

Comment:
Again, we don't really want to use simple terms like `submitted`.
We use the values `pending, submitted, updated, errored` as `status` values in varying resources within our API.

For instance, here in this POST, the claim is going to always receive a `status` value of `pending`. (see line# 39) So it's going to be confusing when we're looking at logs and we see a log that says it was "submitted", but has an actual `status` value of `pending`.

If you want to use the term "submitted" then it should have more context.  Something like, `Submitted to Lighthouse`.  That way, it'll be much easier to tell exactly what is happening in the logs and where that message is coming from in the code.